src/Model/User.php

```diff
@@ -586,7 +586,14 @@ public static function getLanguages(): array
 		$languages = [];
 		$uids      = [];
 
-		$users = DBA::select('user', ['uid', 'language'], ["`verified` AND NOT `blocked` AND NOT `account_removed` AND NOT `account_expired` AND `uid` > ?", 0]);
+		$condition = ["`verified` AND NOT `blocked` AND NOT `account_removed` AND NOT `account_expired` AND `uid` > ?", 0];
```

Comment:
If the value on the `uid` condition isn't dynamic, it's superfluous to use binding.
```suggestion
		$condition = ["`verified` AND NOT `blocked` AND NOT `account_removed` AND NOT `account_expired` AND `uid` > 0"];
```

Reply:
Parameters in SQL should always passed that way, independent if they are constant values or not.